<launch>

    <node pkg="diagnostics" name="voltage_measurements" 
    type="voltage_measurements.py">
    </node>

    <node pkg="diagnostics" name="lamps" 
    type="lamps.py">
    </node>

    <node pkg="diagnostics" name="ir_sensors" 
    type="ir_sensors.py">
    </node>

    <node pkg="diagnostics" name="led_controller" 
    type="led_controller.py" output="screen">
    </node>

    <node pkg="diagnostics" name="broken_lamps_detector" 
    type="broken_lamps_detector.py">
    </node>

</launch>